app/design/adminhtml/default/default/template/sales/order/create/form.phtml

```diff
@@ -36,7 +36,9 @@
         <?php echo $this->getChildHtml('message') ?>
     </div>
     <div id="order-customer-selector" style="display:<?php echo $this->getCustomerSelectorDisplay() ?>">
-        <?php echo $this->getChildHtml('customer') ?>
+        <?php if (!$this->getCustomerId()): ?>
```

Comment:
This condition is an oposite of the commit title.
"don't load customers block if a customer is already set "
Either the commit should read "don't load customers block if a customer is set" or the condition should be chnaged?